### PR TITLE
Chore: Remove replace statements

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

Follow up of https://github.com/grafana/grafana/pull/77734.

I missed the `replace` statement forcing google.golang.org/grpc v1.45.0 (https://nvd.nist.gov/vuln/detail/CVE-2023-44487). This has been removed from `main` with no known issues.

### CODE REVIEW

File: pkg/extensions/main.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ import (
 	_ "github.com/stretchr/testify/require"
 	_ "github.com/vectordotdev/go-datemath"
 	_ "golang.org/x/time/rate"
-	_ "google.golang.org/genproto/googleapis/cloud/kms/v1"
```
Comment (wbrowne) [inline pkg/extensions/main.go:28]: Just checking this isn't causing an issue for enterprise?

Comment (andresmgot) [reply pkg/extensions/main.go:28]: this was removed from `main` with no further comment so I assume no. The library has been deprecated, that's why I had to remove it, the linter complained.

**Review by wbrowne — APPROVED** [review]

**Comment by mdvictor** [comment]
@andresmgot I currently have the same linter issue as you branching from v9.5.x, so with that import removed. I'm trying to update `golang.org/x/net`. I noticed that the build fails when enterprise gets cloned, and passes when it doesn't (as in your second build), it happened to me as well. I don't know if this might affect enterprise in any way. I've linked enterprise locally and ran `make lint-go` but cannot get the same errors as in the pipeline...

**Comment by mdvictor** [comment]
yes, I have these changes in my branch. I'm wondering if there is any issue with just going forward as it is at this point, since it's just a deprecation notice

**Comment by andresmgot** [comment]
seems unrelated so yes, I'd go forward

**Comment by andresmgot** [comment]
Seems to be reproducible just running Drone in the branch 9.5.x in enterprise: https://drone.grafana.net/grafana/grafana-enterprise/63240/4/6

I agree we can add some lint-ignore(s) there since are just deprecating notices. I'll try to send a PR.

**Comment by andresmgot** [comment]
found this PR that we can backport to fix the issue: https://github.com/grafana/grafana-enterprise/pull/6014

**Pull request merged**